addon/package.json

```diff
@@ -62,8 +78,8 @@
     "type": "addon",
     "version": 2,
     "app-js": {
-      "./helpers/page-title.js": "./dist/_app_/helpers/page-title.js",
```

Comment:
these should remain js -- (addon-dev upgrade will handle this)

Reply:
addon dev removed them completely (after I updated to v2)

Reply:
yay!